.github/workflows/prombench.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           "https://api.github.com/repos/$GITHUB_REPOSITORY/statuses/$LAST_COMMIT_SHA"
       - name: Run make deploy to start test
         id: make_deploy
-        uses: docker://prominfra/prombench:master
+        uses: docker://prominfra/prombench:master@sha256:d61131026b316164eb83b19f8611cb9e44de0476b357aa52935d857d813e40d3
```

Comment:
I think it would be best to keep it unpinned here. We want the latest here for prombench (we control this)